blog/report.md

```diff
@@ -0,0 +1,71 @@
+# Benchmark Report
+
+## Goal
+We want to give the user the opportunity to know the minimum size of OCP cluster that can stand for his daily logs written and read from Loki. Our goal is to create a table that specifies for each "T-shirt" size the average and max daily read and write capabilities of Loki.
+
+## Environment
+We ran that on OpenShift running on AWS clusters with configuration:
+
+	apiVersion: v1
+	baseDomain: devcluster.openshift.com
+	compute:
+	- architecture: amd64
+	hyperthreading: Enabled
+	name: worker
+	platform:
+		aws:
+		type: m4.16xlarge
+		rootVolume:
+			type: io1
+			size: 500
+			iops: 8000
+	replicas: 6
+	controlPlane:
+	architecture: amd64
+	hyperthreading: Enabled
+	name: master
+	platform: {}
+	replicas: 3
+
+Running with 64 vCPUs and 256 Memory (GiB)
+
+
+## The Proccess Of Creating The Table
+The benchmark tool helps us observe the data (such as CPU, Memory, etc...) of Loki while sending a different amount of logs each test, the logs are sent by a logger that is deployed when the benchmark test is started (the implementation can be seen at https://github.com/ViaQ/cluster-logging-load-client).
+
+We have been observing mainly the CPU (in MiliCore) and Memory (in GB) as a function of the GiPD (Log load in GB per Day) to see how they change, we ran the benchmark test with different values of GiPD and watched over the CPU and Memory and how the number of Ingesters may affect these values in Loki. First, we ran Loki with 2 Ingesters and then we ran it with 4 Ingesters.
+
+Then a high amount of logs has been sent to Loki by running different values between 2250GB and 6000GB per day (starting with 2250Gb then 2500GB ... and so on until 6000GB). Sending this amount of GB to Loki with 2 Ingesters made the GiPD go too high compared to what has been sent, but the GiPD in Loki with 4 Ingesters was pretty much normal.
+
+
```

Comment:
@jacobsakran I am missing two more sections here ... I think we need:

A section called "conclusions" with:
1. From the results we can learn that .... 
2. The information shows clearly that, and the information is not clear on ... 
etc. 

Also, missing is a "TODO/Next" section that talks about what was done and what is still missing. For example, read tests that we did not do at all ... and for example write tests with different values or different scale of different LokiStack components

Reply:
@jacobsakran can you add another section with TODO/Next that will talk about what is still missing in this evaluation